### DISCUSSION TITLE
Fast forward - audio delay

### DISCUSSION THREAD

**Discussion by swiss-go-to** [SuRGeoNix/Flyleaf#530] [Q&A]
Hi,

I ran into this issue:

When fast forwarding 5x, 6x etc. and then getting back 1x I am getting an Audio delay (1-2 seconds).

I tried various ways to handle it and what somehow worked is:

player.Speed = 1;
player.ReversePlayback = true;
player.ReversePlayback = false;
player.Play();

This handle the Audio delay after changing the speed back 1x.

I would like to know you have ran into this and what standard way you used to handle it.

**Comment by SuRGeoNix** [comment]
@swiss-go-to Can you confirm if you have Config.Audio.FiltersEnabled enabled or not?

**Reply by swiss-go-to** [reply]
Config.Audio.FiltersEnabled is set to **false**

**Reply by SuRGeoNix** [reply]
Try to enable this and see how it goes...
Only in case of MaxLatency you might think about disabling it (haven't tested much for a/v sync on max latency that needs to re-buffer from time to time but it should be fine)

You could also consider reducing audio frames.

**Reply by swiss-go-to** [reply]
It is was better having Config.Audio.FiltersEnabled = true but it results with the Audio coming back right away with a stuttering/chopping audio for 1-2 sec.

I looked into reducing audio frames did not find where to set it?

**Reply by SuRGeoNix** [reply]
Config.Decoder.MaxAudioFrames (default is 10 try to use 3)

**Reply by swiss-go-to** [reply]
Thank you.

I tried with Config.Decoder.MaxAudioFrames = 3 but still stuttering slightly. There is an improvement.
